toolset/setup/linux/languages/elixir.sh

```diff
@@ -1,16 +1,9 @@
 #!/bin/bash
 
-# TODO seems to be broken installation
-echo "WARN: Elixir does not install"
-return 1
+RETCODE=$(fw_exists ${IROOT}/elixir.installed)
+[ ! "$RETCODE" == 0 ] || { return 0; }
 
-# fw_exists v0.13.3.tar.gz
-# [ $? -ne 0 ] || { return 0; }
+fw_depends erlang
+sudo apt-get install -y elixir
```

Comment:
This should be a download, compile, install into `IROOT` instead - we're trying to completely remove sudo